### DISCUSSION TITLE
Theme: how to set height of each line in listview and/or entry+prompt

### DISCUSSION THREAD

**Discussion by fritzrehde** [davatorium/rofi#1725] [Theming]
I want to set the height of each line to 20px, but can't find an option in the `rofi-theme` manpage. Is there such a setting?

**Comment by DaveDavenport** [comment]
its autosized based on content (and equal for every row) .
You can trick it via icons (given the font is lower then 20px).

```
rofi -show drun -theme-str 'element-icon { size: 20px; }'  -no-config -show-icons
```

**Reply by fritzrehde** [reply]
Is this supposed to make the entry + prompt 20px as well? It didn't do that

**Reply by DaveDavenport** [reply]
no, element-icon is not part of entry/prompt. See manpage.

**Reply by DaveDavenport** [reply]
you can also try:
```bash
rofi -show drun -no-config -theme-str 'inputbar,element { padding: calc((20px - 1.0em)/2) 0 ; }'
```
